### DISCUSSION TITLE
How to set the specified GPU and set the number of GPUs in tools/train.py or the corresponding configuration file?

### DISCUSSION THREAD

**Discussion by WinnerMeat** [open-mmlab/mmpose#2608] [Q&A]
How to set the specified GPU and set the number of GPUs in tools/train.py or the corresponding configuration file?
Is there a corresponding setting parameter, similar to gpu_num and the like.

**Comment by lorenz-lb** [comment]
You can set the specified GPU with `CUDA_VISIBLE_DEVICES` before calling your script. 

I am currently also struggling to run the training on multiple GPUs while NOT using the distributed script because I dont have permissions on those things the script does. 

Do you have any advice on how to run your model on multiple gpus?
